src/roktManager.ts

```diff
@@ -9,6 +9,12 @@ export interface IRoktPartnerAttributes {
     [key: string]: string | number | boolean | undefined | null;
 }
 
+export interface IRoktExtensionData {
+    [extensionName: string]: {
+        [key: string]: string | number | boolean | undefined | null;
```

Comment:
Where does the types from this come from?  Are you confident that it doesn't take arrays/dictionaries?